satellite/metainfo/metainfoloop.go

```diff
@@ -0,0 +1,157 @@
+// Copyright (C) 2019 Storj Labs, Inc.
+// See LICENSE for copying information.
+
+package metainfo
```

Comment:
I initially meant the package should be `metainfo/metainfoloop/loop.go`, however having it be `metainfo.Loop` would be cool as well. Let's rename the file to `loop.go`.